core/unit_test/TestTeamBasic.hpp

```diff
@@ -396,6 +394,97 @@ TEST(TEST_CATEGORY, team_broadcast_double) {
   }
 }
 
+struct TeamBroadcastIntPtrFunctor {
+  using TeamMember = typename Kokkos::TeamPolicy<TEST_EXECSPACE>::member_type;
+
+  TeamBroadcastIntPtrFunctor() : view("view") {}
```

Comment:
I would do
```suggestion
  Kokkos::View<int, TEST_EXECSPACE> view {"lbl"};
```
and not bother with the default constructor.